panda/src/display/CMakeLists.txt

```diff
@@ -90,7 +90,9 @@ composite_sources(p3display P3DISPLAY_SOURCES)
 add_component_library(p3display SYMBOL BUILDING_PANDA_DISPLAY
   ${P3DISPLAY_HEADERS} ${P3DISPLAY_SOURCES})
 target_link_libraries(p3display p3cull p3pgraphnodes)
-target_use_packages(p3display X11)
+if(HAVE_X11)
+  target_use_packages(p3display X11)
+endif()
```

Comment:
This is redundant, since `target_use_packages` actually already performs this very check.

If X11 headers are used by p3display when HAVE_X11 is on (which leaves a sour taste in my mouth for other reasons), the best thing to do would be to leave it as you had it.

Reply:
Hah, shows how much I know about cmake. I'll change that back.

Reply:
No worries - that's actually a macro defined in PackageConfig.cmake and not part of CMake itself. :)